src/compiler/output-targets/output-angular.ts

```diff
@@ -102,19 +114,9 @@ export class ${tagNameAsPascal} {`];
   lines.push(`  constructor(c: ChangeDetectorRef, r: ElementRef, protected z: NgZone) {
     c.detach();
     this.el = r.nativeElement;`);
-  if (hasOutputs) {
```

Comment:
I think outputs still need to be here, in the constructor, since they apply to the instance, adding own properties, not to the prototype.